src/App.js

```diff
@@ -18,34 +21,80 @@ function App() {
 
   return (
     <AppWrapper>
-      {isEditFormShown ? 
-        <SaveGameForm 
-            onSubmit={addGameProfile} 
-            isEditFormShown={isEditFormShown} 
-            targetProfile={targetProfile} 
-            editGameProfile={editGameProfile}
-            cancelEditModus={cancelEditModus}/>
-      : 
-      <SaveGameFormWrapper>
-        <SaveGameForm onSubmit={addGameProfile} />
-        <GameCardsList 
-            savedGameProfiles={savedGameProfiles} 
-            onDelete={deleteGameProfile} 
-            onEdit={prepareEditModus}/>
-      </SaveGameFormWrapper>
-      }
+      <HeaderWrapper>
+        <Header/>
+      </HeaderWrapper>
```

Comment:
Maybe you could refractor this into one component/file ?

Reply:
I separated it, in order to have the styles, which come from the parent element, not in styleguidist. ``<HeaderWrapper>`` declares the fixed position for example and that would not work in styleguidist. :)